src/main/java/g2401_2500/s2456_most_popular_video_creator/Solution.java

```diff
@@ -0,0 +1,37 @@
+package g2401_2500.s2456_most_popular_video_creator;
+
+// #Medium #Array #String #Hash_Table #Sorting #Heap_(Priority_Queue)
```

Comment:
Please update generatemd tool with latest version. It should generate tags without ().